### DISCUSSION TITLE
Drive Details GpxExport file unable to display correct speed in Google earth

### DISCUSSION THREAD

**Discussion by zjh02249** [teslamate-org/teslamate#1838] [Q&A]
I export the gpx file then import into the googleEarth ,but The track of the vehicle indicated that the speed was much higher than Browser speed
<img width="854" alt="WeChatc673b969b06988276f5262f075ecf198" src="https://user-images.githubusercontent.com/18898962/125035700-b7fdaf80-e0c4-11eb-851a-82bd1f479cdf.png">
<img width="396" alt="WeChatbe6776d0f0339a5e14a8ea8c74177d94" src="https://user-images.githubusercontent.com/18898962/125035709-ba600980-e0c4-11eb-872a-5ed56d9ede60.png">

- TeslaMate version:1.23.6
- Type of installation: Docker 
- OS TeslaMate is installed on:aliyunOS
- User OS & Browser:User OS
- Others:

**Comment by adriankumpf** [comment]
The GPX file actually only contains information about positions, elevations and timestamps. So it's Google Earth that tries to estimate the speed. It probably fails to accurately calculate the speed because of the accuracy of elevation/GPS information. 

We could include the speed in the file but it's not part of the GPX 1.1 schema and therefore would not conform to the standard. I'm not sure on the implications that would have.

**Comment by jim0266** [comment]
Hello, this still does not work for Google Maps. When I try to import a Teslamate GPX file into mymaps.google.com I get the following error:

"The file contains invalid or unsupported data, or the file is too large to parse."

The files I tried were only 550kb and 1.1MB.
